### PR TITLE
Update Installation Instructions for VS Code .NET MAUI Extension

### PR DESCRIPTION
**Pull request by keylime-unicorn**
- Updated the instructions for installing the .NET MAUI Extension for VS Code including all dependencies, workloads, and target platforms. 
- Added clarification for setting up android as a target platform on all operating systems. 

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [docs/get-started/installation.md](https://github.com/dotnet/docs-maui/blob/7a43d382992b9ab13dcd0c895f12c3343c0f35fa/docs/get-started/installation.md) | [Installation](https://review.learn.microsoft.com/en-us/dotnet/maui/get-started/installation?branch=pr-en-us-2033) |

### CODE REVIEW

**Comment by keylime-unicorn** [comment]
@BethMassi @maddymontaquila - Could you please review and give feedback on this PR? Thank you!

**Comment by keylime-unicorn** [comment]
@jonpryor - Could you please review as well? Thank you!

**Review by BethMassi — APPROVED** [review]

File: docs/get-started/installation.md [hunk 1]
```diff
@@ -111,52 +111,41 @@ To install Visual Studio 2022 for Mac behind a firewall, certain endpoints must
 > [!NOTE]
 > The .NET MAUI extension requires the C# Dev Kit and prerelease C# extensions. You must sign into C# Dev Kit to use the .NET MAUI extension's functionality. See [this blog post](https://devblogs.microsoft.com/visualstudio/announcing-csharp-dev-kit-for-visual-studio-code/) for more information about C# Dev Kit and its family of extensions.
 
-## Set up .NET, .NET MAUI, and target platforms
+## Install .NET and .NET MAUI workloads
 
-To develop with .NET MAUI in Visual Studio code, you need to have .NET, the .NET MAUI workload, and a valid target platform based on your development machine's operating system:
+1. Install [.NET 8](/dotnet/core/install/windows).
+    1. On Windows, it is recommended to use the Visual Studio installer to manage .NET and the .NET MAUI workload installations. Instructions on using the Visual Studio installer can be found [here](./installation.md?tabs=vswin).
```
Comment (davidbritch) [inline docs/get-started/installation.md]:
```suggestion
    On Windows, it's recommended to use the Visual Studio installer to manage .NET and the .NET MAUI workload installations. Instructions on using the Visual Studio installer can be found [here](./installation.md?tabs=vswin).
```

File: docs/get-started/installation.md [hunk 2]
```diff
@@ -111,52 +111,41 @@ To install Visual Studio 2022 for Mac behind a firewall, certain endpoints must
...
+1. Install .NET MAUI workload
```
Comment (davidbritch) [inline docs/get-started/installation.md]:
```suggestion
1. Install the .NET MAUI workload:
```

File: docs/get-started/installation.md [hunk 3]
```diff
@@ -111,52 +111,41 @@ To install Visual Studio 2022 for Mac behind a firewall, certain endpoints must
...
+    - Windows/MacOS: Run `dotnet workload install maui` in a terminal.
```
Comment (davidbritch) [inline docs/get-started/installation.md]:
```suggestion
    On Windows and macOS, run the following command in a terminal: 
    
    ```cmd
    dotnet workload install maui
    ```
```

File: docs/get-started/installation.md [hunk 4]
```diff
@@ -111,52 +111,41 @@ To install Visual Studio 2022 for Mac behind a firewall, certain endpoints must
...
+    - Linux: Run `dotnet workload install maui-android` in a terminal.
```
Comment (davidbritch) [inline docs/get-started/installation.md]:
```suggestion
    On Linux, run the following command in a terminal:
    
    ```cmd
    dotnet workload install maui-android
    ```
```

File: docs/get-started/installation.md [hunk 5]
```diff
@@ -111,52 +111,41 @@ To install Visual Studio 2022 for Mac behind a firewall, certain endpoints must
...
+
+## Set up Target Platforms
```
Comment (davidbritch) [inline docs/get-started/installation.md]:
```suggestion
## Set up target platforms
```

File: docs/get-started/installation.md [hunk 6]
```diff
@@ -111,52 +111,41 @@ To install Visual Studio 2022 for Mac behind a firewall, certain endpoints must
...
+
+To debug a .NET MAUI app in Visual Studio code, you need to have a valid target platform based on your development machine's operating system:
```
Comment (davidbritch) [inline docs/get-started/installation.md]:
```suggestion
To debug a .NET MAUI app in Visual Studio Code, you need to have a valid target platform for your development machine's operating system:
```

File: docs/get-started/installation.md [hunk 7]
```diff
@@ -111,52 +111,41 @@ To install Visual Studio 2022 for Mac behind a firewall, certain endpoints must
...
 
 | Your Operating System | Supported Target Platforms |
 |---|---|
 | Windows | Windows, Android |
 | macOS | Android, iOS, macOS |
 | Linux | Android |
 
-### Windows
+### iOS and MacOS
```
Comment (davidbritch) [inline docs/get-started/installation.md]:
```suggestion
### iOS and macOS
```

File: docs/get-started/installation.md [hunk 8]
```diff
@@ -111,52 +111,41 @@ To install Visual Studio 2022 for Mac behind a firewall, certain endpoints must
...
 
-The recommended approach to setup your Windows machine for .NET MAUI development is through the Visual Studio installer. You can follow the instructions in the Visual Studio tab on this page to walk through installing the .NET MAUI workload. You can also use Visual Studio to manage different versions of Android, create emulators, update .NET and .NET MAUI, and more.
+To debug to an iOS or MacOS target in Visual Studio Code, follow the steps below:
```
Comment (davidbritch) [inline docs/get-started/installation.md]:
```suggestion
To debug to an iOS or macOS target in Visual Studio Code:
```

File: docs/get-started/installation.md [hunk 9]
```diff
@@ -111,52 +111,41 @@ To install Visual Studio 2022 for Mac behind a firewall, certain endpoints must
...
-    On Windows, the suggested Android SDK directory value is `%LOCALAPPDATA%/Android/Sdk`.
+1. Install the [latest stable Xcode](https://apps.apple.com/us/app/xcode/id497799835?mt=12).
+1. Run `xcode-select --install` in a terminal to acquire the Xcode command line tools.
 
-> [!NOTE]
-> See the [Troubleshooting > Android Setup](#android-setup) section for more information on setting up the Android SDK.
+### Android
 
-### macOS
+To debug to an Android target in Visual Studio Code, follow the steps below:
```
Comment (davidbritch) [inline docs/get-started/installation.md]:
```suggestion
To debug to an Android target in Visual Studio Code:
```

File: docs/get-started/installation.md [hunk 10]
```diff
@@ -111,52 +111,41 @@ To install Visual Studio 2022 for Mac behind a firewall, certain endpoints must
...
-
-### Linux
-
-1. Install [.NET 8](/dotnet/core/install/linux).
-1. After .NET has installed, run `dotnet workload install maui-android` in a terminal.
-1. If you'd like to debug to Android in Visual Studio Code, you also have to:
-    1. Install [Microsoft OpenJDK 11](/java/openjdk/download#openjdk-11).
-    1. Install the Android SDK from your preferred package manager, or from the [Android website](https://developer.android.com/about/versions/13/setup-sdk).
+1. Install [Microsoft OpenJDK 11](/java/openjdk/download#openjdk-11).
+1. Install the Android SDK via one of the following options:
```
Comment (davidbritch) [inline docs/get-started/installation.md]:
```suggestion
1. Install the Android SDK via one of the following approaches:
```

File: docs/get-started/installation.md [hunk 11]
```diff
@@ -166,23 +155,17 @@ You might face issues when setting up the .NET MAUI extension for Visual Studio
...
 
-### Android setup
-
-There are multiple ways you can set up and manage Android SDKs on your machine:
-
-* (Recommended) Use the [`dotnet build -t:InstallAndroidDependencies` target](#using-installandroiddependencies).
-* Install via Visual Studio (Windows only).
-* Install via Android Studio.
-* Use the [Android Command Line Tools](https://developer.android.com/studio#command-tools).
+#### Using "InstallAndroidDependencies"
```
Comment (davidbritch) [inline docs/get-started/installation.md]:
```suggestion
#### Using the InstallAndroidDependencies target
```

File: docs/get-started/installation.md [hunk 12]
```diff
@@ -111,52 +111,41 @@ To install Visual Studio 2022 for Mac behind a firewall, certain endpoints must
...
-### Linux
-
-1. Install [.NET 8](/dotnet/core/install/linux).
-1. After .NET has installed, run `dotnet workload install maui-android` in a terminal.
-1. If you'd like to debug to Android in Visual Studio Code, you also have to:
-    1. Install [Microsoft OpenJDK 11](/java/openjdk/download#openjdk-11).
-    1. Install the Android SDK from your preferred package manager, or from the [Android website](https://developer.android.com/about/versions/13/setup-sdk).
+1. Install [Microsoft OpenJDK 11](/java/openjdk/download#openjdk-11).
+1. Install the Android SDK via one of the following options:
+    * **(Recommended)** Create a new .NET MAUI project (`dotnet new maui`) and use the [InstallAndroidDependencies target](#using-installandroiddependencies).
```
Comment (davidbritch) [inline docs/get-started/installation.md]:
```suggestion
    * **(Recommended)** Create a new .NET MAUI project (`dotnet new maui`) and use the [InstallAndroidDependencies target](#using-the-installandroiddependencies-target).
```

File: docs/get-started/installation.md [hunk 13]
```diff
@@ -166,23 +155,17 @@ You might face issues when setting up the .NET MAUI extension for Visual Studio
...
-* Install via Visual Studio (Windows only).
-* Install via Android Studio.
-* Use the [Android Command Line Tools](https://developer.android.com/studio#command-tools).
+#### Using "InstallAndroidDependencies"
 
-No matter which way you install Android, you can develop .NET MAUI apps in Visual Studio Code. If the following troubleshooting steps don't solve your issue, please [Report a Problem](https://github.com/microsoft/vscode-dotnettools/issues).
+.NET 8 has a build target that helps set up your Android environment for you. Run the following command in a terminal to configure your machine and set up your Android environment:
 
-#### Using "InstallAndroidDependencies"
+> `dotnet build -t:InstallAndroidDependencies -f:net8.0-android -p:AndroidSdkDirectory "<AndroidSdkPath>" -p:JavaSdkDirectory "<JavaSdkPath>" -p:AcceptAndroidSDKLicenses=True`
```
Comment (davidbritch) [inline docs/get-started/installation.md]:
```suggestion
```cmd
dotnet build -t:InstallAndroidDependencies -f:net8.0-android -p:AndroidSdkDirectory "<AndroidSdkPath>" -p:JavaSdkDirectory "<JavaSdkPath>" -p:AcceptAndroidSDKLicenses=True
```

File: docs/get-started/installation.md [hunk 14]
```diff
@@ -166,23 +155,17 @@ You might face issues when setting up the .NET MAUI extension for Visual Studio
...
 
-* .NET 8 has a build target that helps set up your Android environment for you. You can add or remove the following properties to `dotnet build -t:InstallAndroidDependencies -f:net8.0-android` to configure your machine:
-  * `-p:AndroidSdkDirectory "<PATH>"` installs or updates Android dependencies to the specified path (Note: You must use an absolute path without a tilde "~").
-  * `-p:JavaSdkDirectory "<PATH>"` installs Java to the specified path (Note: You must use an absolute path without a tilde "~").
-  * `-p:AcceptAndroidSDKLicenses=True` accepts the necessary Android licenses for development.
+* `AndroidSdkDirectory "<AndroidSdkPath>"` : installs or updates Android dependencies to the specified path (Note: Must use an absolute path).
```
Comment (davidbritch) [inline docs/get-started/installation.md]:
```suggestion
In the command above:
* `AndroidSdkDirectory "<AndroidSdkPath>"`: installs or updates Android dependencies to the specified absolute path.
```

File: docs/get-started/installation.md [hunk 15]
```diff
@@ -166,23 +155,17 @@ You might face issues when setting up the .NET MAUI extension for Visual Studio
...
+  * Windows: the suggested AndroidSdkPath is `%LOCALAPPDATA%/Android/Sdk`.
+  * MacOS: the suggested AndroidSdkPath is `$HOME/Library/Android/sdk`.
+* `JavaSdkDirectory "<JavaSdkPath>"` : installs Java to the specified path (Note: Must be an absolute path).
```
Comment (davidbritch) [inline docs/get-started/installation.md]:
```suggestion
* `JavaSdkDirectory "<JavaSdkPath>"`: installs Java to the specified absolute path.
```

File: docs/get-started/installation.md [hunk 16]
```diff
@@ -166,23 +155,17 @@ You might face issues when setting up the .NET MAUI extension for Visual Studio
...
+* `AcceptAndroidSDKLicenses=True` : accepts the necessary Android licenses for development.
```
Comment (davidbritch) [inline docs/get-started/installation.md]:
```suggestion
* `AcceptAndroidSDKLicenses=True`: accepts the required Android licenses for development.
```

**Review by davidbritch — CHANGES_REQUESTED** [review]
All minor nit picks :)

**Review by davidbritch — APPROVED** [review]

**Pull request merged**